sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientBuilderTest.java

```diff
@@ -81,6 +85,33 @@ public void clientBuilderWithDefaultPipeline(HttpClient httpClient, FormRecogniz
             assertNotNull(clientBuilder.buildClient().beginRecognizeContentFromUrl(input).getFinalResult()));
     }
 
+    /**
+     * Test for invalid endpoint, which throws connection refused exception message.
+     */
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.formrecognizer.TestUtils#getTestParameters")
+    public void clientBuilderWithInvalidEndpoint(HttpClient httpClient, FormRecognizerServiceVersion serviceVersion) {
```

Comment:
Are you covering this as part of a different PR?

Reply:
yes. covered in this PR. Speed up to around 3mins now